_dev/Makefile

```diff
@@ -46,9 +46,9 @@ $(BUILD_DIR)/$(OS_ARCH)/.ok_$(VER_OS_ARCH): clean
 	mkdir -p "$(GOROOT_LOCAL)/pkg"
 
 # Copy src/tools from system GOROOT
-	cp -Hr $(GOROOT_ENV)/src $(GOROOT_LOCAL)/src
```

Comment:
why this change?

Reply:
See the first commit message:
* Options `-r` and `-R` are equivalent for GNU sed, but macOS requires
  `-R' or else it fails with:
  cp: the -H, -L, and -P options may not be specified with the -r option.

Reply:
I know, but I would like this comment in the code (btw: I don't understand the commit message - it says something about "sed", while change is done to "cp")

Reply:
I'll add the comment in the Makefile, but I think such details are not important to comment about (once it works, it will probably not be changed).

I'll fix the commit message, it should indeed have said cp. Thanks!